src/main/java/com/github/dockerjava/api/model/SELContext.java

```diff
@@ -0,0 +1,44 @@
+package com.github.dockerjava.api.model;
+
+/**
+ * The context mode of bound volume in SELinux.
+ * Host path <code>shared</code> - can be used by all containers, <code>private</code> - by only this container
+ *
+ * @see http://www.projectatomic.io/blog/2015/06/using-volumes-with-docker-can-cause-problems-with-selinux/
+ */
+public enum SELContext {
```

Comment:
sounds like it may limit extensibility in future (i.e. some new field that user will implement in his own code), wouldn't it better to use abstract class instead enum?